tests/Integration/Behaviour/Features/Scenario/Cart/Adding/Product/add_combination.feature

```diff
@@ -28,3 +31,16 @@ Feature: Add product combination in cart
     When I add 600 items of combination "combi1" of product "product7"
     Then I should have 600 items of combination "combi1" of product "product7" in my cart
     Then the remaining available stock for combination "combi1" of product "product7" should be -100
+
+  @add-combinations-to-cart
```

Comment:
@matthieu-rolland I would add a behat test with only 2 items (in this case, the specific price doesn't apply)

Reply:
thanx, done 

 I did it in the same scenario so that it follows the real use case:  first I test the price with only 2 products (it must not change), then I add one product, then I check again the price of each product, the quantity price rule must be applied.